xoto3/dynamodb/write_versioned/specify.py

```diff
@@ -60,7 +65,12 @@ def presume(
             tables={
                 **transaction.tables,
                 table_name: _TableData(
-                    items={**table_data.items, hkey: item_value},
+                    items={
+                        **table_data.items,
+                        hkey: dynamodb_prewrite(item_value, prewrite_transform)
```

Comment:
this line is the main change. apply the prewrite to any item supplied by the user rather than fetched directly from DynamoDB.